src/helpers/error-message.helper.ts

```diff
@@ -23,33 +23,55 @@ export class ErrorMessageHelper {
 
     const kind = error.kind;
     let message: string;
+    let title = "default title";
```

Comment:
Zmieniłbym tytuł na polski, gdyby jednak czasem się wyświetlił to mógłby wywołać konsternację Pani Iwonki :) 
Coś w stylu: "Nie rozpoznano błędu"